src/components/data-processing/DoubletScores/CalculationConfig.jsx

```diff
@@ -1,40 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {
-  Slider,
-  Form,
-} from 'antd';
+import { Form } from 'antd';
 
-import BandwidthOrBinstep from '../ReadAlignment/PlotStyleMisc';
+import SliderWithInput from '../../SliderWithInput';
 
 const DoubletScoresConfig = (props) => {
   const {
-    config, disabled, plotType, updateSettings,
+    config, disabled, updateSettings,
   } = props;
 
-  const filtering = false;
```

Comment:
I am not sure about the behaviour of this thing, it depends on `filtering` which is always false so I don't really see what it is intended for.

Removing it doesn't seem to have any effect on the behaviour of the app.

Reply:
We also have it for CellSizeDistribution and MitochondrialContent. In all three cases, git blame points to @cosa65 , so you should ask him :-D

Reply:
Upon further research, looks very much like the result of removing a `filtering` property https://github.com/biomage-ltd/ui/blob/0b50bbecf8eb7d7550c2ee887618670cf5e3faee/src/components/data-processing/DoubletScores/DoubletScores.jsx#L268

Can you also remove this code from the other two filters I mentioned?

Reply:
I think I removed in the other places already as far as I can see)